src/modules.py

```diff
@@ -447,7 +451,236 @@ def parse_panda(config: Dict[str, Any]) -> IocCollection:
+            iocs.add_rsa_key(RsaKey(int(key["n"]), int(key["e"])))
+
+    return iocs
+
+
+@module("vjworm")
+def parse_vjworm(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+    for urls in config.get("urls", []):
+        iocs.try_add_url(urls["url"])
+    return iocs
+
+
+@module("nymaim")
+def parse_nymaim(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+    if "encryption_key" in config:
+        iocs.add_key("encryption_key", config["encryption_key"])
+    if "public_key" in config:
+        key = config["public_key"]
+        iocs.add_rsa_key(RsaKey(int(key["n"]), int(key["e"])))
+    for url in config["urls"]:
+        iocs.try_add_url(url)
+    return iocs
+
+
+@module("globeimposter")
+def parse_globeimposter(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+    for email in config.get("emails", []):
+        iocs.add_email(email)
+    if "ransom_message" in config:
+        iocs.add_ransom_message(config["ransom_message"])
+    if "urls" in config:
+        for url in config["urls"]:
+            iocs.try_add_url(url)
+    return iocs
+
+
+@module("gootkit")
+def parse_gootkit(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+    for domain in config.get("domains", []):
+        iocs.try_add_url(domain["cnc"])
+    return iocs
+
+
+@module("hancitor")
+def parse_hancitor(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+    for urls in config.get("urls", []):
+        iocs.try_add_url(urls["url"])
+    return iocs
+
+
+@module("zeus")
+def parse_zeus(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+    if "rc4sbox" in config:
+        iocs.add_key("rc4sbox", config["rc4sbox"])
+    if "cnc" in config:
+        iocs.try_add_url(config["cnc"])
+    if "urls" in config:
+        for url in config["urls"]:
+            iocs.try_add_url(url)
+    return iocs
+
+
+@module("vmzeus")
+def parse_vmzeus(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+    if "rc4sbox" in config:
+        iocs.add_key("rc4sbox", config["rc4sbox"])
+    if "rc4sbox" in config:
+        iocs.add_key("rc6sbox", config["rc6sbox"])
+    for url in config["urls"]:
+        iocs.try_add_url(url)
+    return iocs
+
+
+@module("sendsafe")
+def parse_sendsafe(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+    iocs.try_add_network_location(host=config["cnc"], port=int(config["http_port"]))
+    return iocs
+
+
+@module("necurs")
+def parse_necurs(config: Dict[str,Any]) -> IocCollection:
+    iocs = IocCollection()
+    for domain in config.get("domains", []):
+        iocs.try_add_url(domain["cnc"])
+    if "c2_public_key" in config:
+        key = config["c2_public_key"]
+        if type(key) == dict:
+            iocs.add_rsa_key(RsaKey(int(key["n"]), int(key["e"])))
+        elif type(key) == str:
+            iocs.add_key("c2_public_key", config["c2_public_key"])
```

Comment:
since it's an RSA key, you should add it as such. would try_add_rsa_from_pem method work here?